### ISSUE TITLE
Verify if subnet restrictions still apply

### ISSUE THREAD

**Issue by ineiti** [dedis/onet#671]
While reviewing #665 I saw the following:

https://github.com/dedis/onet/blame/master/simul/platform/MININET.md#L24

```
Unfortunately we still have the restriction that _all servers need to be on the
same subnet_! We know about the situation and hope to have a solution ready
sometime in the near future.
```

But I thought that https://github.com/dedis/onet/blob/master/simul/platform/mininet/start.py#L48 takes care of this. Can somebody confirm this?

If this restriction has been removed, I'll adjust the `MININET.md` file.

**Comment by jeffallen** [comment]
I tested #665 with this server_list:

```
$ cat server_list 
iccluster076.iccluster.epfl.ch
iccluster079.iccluster.epfl.ch
iccluster080.iccluster.epfl.ch
iccluster105.iccluster.epfl.ch
```

resulting in this deploy/list:

```
CountTest Ed25519 1000 50
2 false false true
show_date.sh
10.90.40.2 10.1.0.0/16 2
10.90.40.5 10.2.0.0/16 2
10.90.40.6 10.3.0.0/16 2
10.90.36.35 10.4.0.0/16 1
```

Note that the 4th machine is on a different subnet than the rest.

The ipip tunnels seemed to be created correctly, and the simulation ran to completion. I did need to fix some errors to make this work correctly, those fixes will be in #665.

**Comment by ineiti** [comment]
Thanks a lot for testing this! I'm glad it worked out after some hacking around...
